### PR TITLE
fix: fixed some deprecated pages and upgrade

### LINKED ISSUE #186: BUGS: Fix the deprecated pages and upgrade accordingly
When doing `pnpm install`

```
packages/ui                              |  WARN  deprecated @mui/core@5.0.0-alpha.54
apps/feedahelp                           |  WARN  deprecated @storybook/addon-knobs@3.4.12
 WARN  deprecated csurf@1.11.0: Please use another csrf package
apps/feedahelp                           |  WARN  deprecated extract-text-webpack-plugin@3.0.2
apps/feedahelp                           |  WARN  deprecated popper-max-size-modifier@0.2.0
apps/feedahelp                           |  WARN  deprecated uuid@3.4.0
apps/feedahelp                           |  WARN  deprecated postcss-cssnext@3.1.1
apps/feedahelp                           |  WARN  deprecated request@2.88.2
apps/feedahelp                           |  WARN  deprecated acorn-dynamic-import@2.0.2
apps/feedahelp                           |  WARN  deprecated core-js@2.6.12
Downloading registry.npmjs.org/@swc/core-darwin-arm64/1.3.78: 12.8 MB/12.8 MB, done
apps/feedahelp                           |  WARN  deprecated core-js@1.2.7
apps/feedahelp                           |  WARN  deprecated querystring@0.2.1
apps/feedahelp                           |  WARN  deprecated browserslist@2.11.3
apps/feedahelp                           |  WARN  deprecated array.prototype.flatten@1.2.1
apps/feedahelp                           |  WARN  deprecated har-validator@5.1.5
apps/feedahelp                           |  WARN  deprecated chokidar@2.1.8
apps/feedahelp                           |  WARN  deprecated fsevents@1.2.13
apps/feedahelp                           |  WARN  deprecated flatten@1.0.3
apps/feedahelp                           |  WARN  deprecated browserslist@1.7.7
apps/feedahelp                           |  WARN  deprecated svgo@0.7.2
apps/feedahelp                           |  WARN  deprecated svgo@1.3.2
apps/feedahelp                           |  WARN  deprecated source-map-resolve@0.5.3
apps/feedahelp                           |  WARN  deprecated stable@0.1.8
apps/feedahelp                           |  WARN  deprecated resolve-url@0.2.1
apps/feedahelp                           |  WARN  deprecated source-map-url@0.4.1
apps/feedahelp                           |  WARN  deprecated urix@0.1.0
```

### PR DESCRIPTION
**Pull request by Rakibs511**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #186 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/feedahelp/package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "react-hook-form": "^7.45.3",
     "react-hot-toast": "^2.4.1",
     "recoil": "^0.7.7",
-    "styled-components": "^6.0.7",
+    "styled-components": "^5.2.0",
```
Comment (TanvirAlam) [inline apps/feedahelp/package.json:43]: you downgraded this, but why?

File: packages/ui/package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
   "dependencies": {
     "@emotion/styled": "^11.11.0",
     "@jest/globals": "^29.5.0",
-    "@mui/core": "5.0.0-alpha.54",
+    "@mui/base": "5.0.0-beta.11",
```
Comment (TanvirAlam) [inline packages/ui/package.json]: you replaced `core` with `base`, something fishy. it smell like I farted. something smells good or bad I dont kknow

**Review by TanvirAlam — CHANGES_REQUESTED** [review]
word?

File: apps/feedahelp/package.json (hunk 1 above)
Comment (Rakibs511) [reply apps/feedahelp/package.json:43]: cz next-sanity package showed a warning to downgrade that

File: packages/ui/package.json (hunk 2 above)
Comment (Rakibs511) [reply packages/ui/package.json]: sir core package is deprecated and suggests using base

**Review by TanvirAlam — CHANGES_REQUESTED** [review]
do the changes

File: apps/feedahelp/package.json (hunk 1 above)
Comment (TanvirAlam) [reply apps/feedahelp/package.json:43]: ok
lets see

**Review by TanvirAlam — APPROVED** [review]
so far so good

**Pull request merged**